ble_controller/Kconfig

```diff
@@ -54,12 +62,13 @@ config BLE_CONTROLLER_S132
 	select BT_CTLR_ADV_EXT_SUPPORT
 
 config BLE_CONTROLLER_S140
-	depends on (SOC_NRF52811 || SOC_NRF52840)
+	depends on (SOC_NRF52811 || SOC_NRF52840 || SOC_SERIES_NRF53X)
```

Comment:
Why not SOC_NRF5340?

Reply:
Even though there is only nRF5340 now, there should be no reason that the controller shouldn't work with any future series of nRF53 :) If there are special cases, we'll handle it then. The reason specific versions of the nRF52-series is used here is because of HW compability.